cps/tasks/metadata_extract.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, task_message, media_url, original_url, current_user_name):
         self.original_url = self._format_original_url(original_url)
         self.is_playlist = None
         self.current_user_name = current_user_name
-        self.start_time = self.end_time = datetime.now()
+        self.start_time = self.end_time = datetime.now(timezone.utc)
```

Comment:
This is already being done in `super(...).__init__(...)`, the constructor of CalibreTask.

Reply:
@deldesir do you agree?  (Do revise if so!)

Reply:
This is the reason I made the previous change to consider the time set in the constructor.

Reply:
> This is the reason I made the previous change to consider the time set in the constructor.

@deldesir Plz clarify the exact line number to help me understand? Thx!

Reply:
[Line #26](https://github.com/iiab/calibre-web/blob/9f52798b4d068bc7f804945f4a364b2650723f2c/cps/tasks/metadata_extract.py#L26) for metadata_extract.py

[Line #35](https://github.com/iiab/calibre-web/blob/9f52798b4d068bc7f804945f4a364b2650723f2c/cps/tasks/download.py#L35) for download.py